packages/react/src/styles/uploads-list.css

```diff
@@ -0,0 +1,64 @@
+.w3-uploads-list {
+  margin-bottom: 1.25rem;
+}
+
+.w3-uploads-list-no-uploads {
+  color: rgb(212 212 216);
+}
+
+.w3-uploads-list-data {
+  overflow: auto;
+  border-radius: 0.375rem;
+  border-width: 1px;
+  border-color: rgb(209 213 219);
+  border-style: solid;
+}
+
+.w3-uploads-list table {
+  width: 100%;
+  border-collapse: collapse;
+}
+
+.w3-uploads-list table>*+* {
+  border-top-width: 1px;
+  border-bottom-width: 0px;
+  border-color: rgb(209 213 219);
+}
+
+.w3-uploads-list thead {
+  text-align: left;
+  font-size: 0.75rem;
+  line-height: 1rem;
+  letter-spacing: 0.025em;
+  color: rgb(212 212 216);
+}
+
+.w3-uploads-list th {
+  padding: 0.75rem;
+}
+
+.w3-uploads-list td {
+  display: block;
+  width: 16rem;
+  padding: 0.5rem;
+  padding-left: 0.75rem;
+  font-family: ui-monospace, SFMono-Regular, Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace;
+  font-size: 0.75rem;
+  line-height: 1rem;
+}
+
+.w3-uploads-list td a {
+  text-decoration: none;
+  color: white;
+}
+
+.w3-uploads-list nav {
+  display: flex;
+  flex-direction: row;
+  margin-top: 1rem;
+  margin-bottom: 1rem;
+}
+
+.w3-uploads-list nav>*+*{
```

Comment:
i have never seen one like that before!

Reply:
right? cribbed from the tailwind docs! eg, https://tailwindcss.com/docs/space